src/sailfishusermanager.cpp

```diff
@@ -332,6 +334,26 @@ uint SailfishUserManager::addUser(const QString &name)
     return addSailfishUser(user, name);
 }
 
+int SailfishUserManager::creationScriptsUserFiles(uint uid)
+{
+    // Execute user creation scripts
+    QDir creation(USER_CREATE_SCRIPT_DIR, "*.sh", QDir::NoSort, QDir::Files | QDir::Executable);
+
+    auto entryList = creation.entryList();
+
+    QCollator collator;
+    collator.setNumericMode(true);
+    collator.setCaseSensitivity( Qt::CaseInsensitive );
```

Comment:
I think this probably introduces some non-determinism here. So if two files only differ by case, they can be returned by `entryList()` in either order and if this is stable sort, then it can vary between devices. Unlikely case but possible and undesired. Also if the sort is not stable, then it doesn't really matter since it's undesired anyway. I would say we want this to be case sensitive so that all names are different from each other.